tests/ceph_installer/test_ansible.py

```diff
@@ -144,6 +153,17 @@ def run(ceph_cluster, **kw):
         LOG.error("Failed during deployment")
         return rc
 
+    # Check logs has generated after running the deploy playbook
+    if custom_log:
```

Comment:
Shouldn't we also pass the custom log option to the ansible playbook?

Reply:
@psathyan, sorry I did not get you! Can you please explain it?

Reply:
Sorry for the confusion...

By default the `log_path` is set to `None`, hence shouldn't we modify the ansible.cfg variable `DEFAULT_LOG_PATH` when `custom_log` is set to True?

Reply:
> Sorry for the confusion...
> 
> By default the `log_path` is set to `None`, hence shouldn't we modify the ansible.cfg variable `DEFAULT_LOG_PATH` when `custom_log` is set to True?

@psathyan, I believe that's not true, the default `log_path` will never be set to 'None'! Also, the test case is not meant to modify the default `log_path` but to look into the `log_path` and check if the logs are available. 

Let me know if I'm still missing something here! 

As per my knowledge the default `log_path` will be set to: 
``` log_path = $HOME/ansible/ansible.log ```

Reply:
I don't have a 4.x setup to verify but going by https://docs.ansible.com/ansible/latest/reference_appendices/logging.html it is set to None.

Reply:
@psathyan, we are discussing the `ceph-ansible`! Our package is designed with the default configuration when you'll install the ceph-ansible rpm package `ansible.cfg` will get created at `/usr/share/ceph-ansible` with the default log location `$HOME/ansible/ansible.log` until and unless you change it to other location.

I hope this helps! :-)

Reply:
Thanks for clarifying it.